clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -3489,9 +3506,26 @@ public void consumerBatchUnsubscribe(Set<PubSubTopicPartition> topicPartitionSet
 
   public abstract void consumerUnSubscribeAllTopics(PartitionConsumptionState partitionConsumptionState);
 
-  public void consumerSubscribe(PubSubTopicPartition pubSubTopicPartition, long startOffset, String kafkaURL) {
-    // TODO: pass special format of kafka url as input here when subscribe to the separate incremental push topic
+  /**
+   * This method will try to resolve actual topic-partition from input Kafka URL and subscribe to the resolved
+   * topic-partition.
+   */
+  public void consumerSubscribe(
+      PubSubTopic pubSubTopic,
+      PartitionConsumptionState partitionConsumptionState,
```

Comment:
I think either way should be ok. For now, all the places calling this method has PCS available, so I prefer for now let's keep using PCS in the method signature, otherwise it will result in a bunch of changes in files, which can potentially be done in a separate PR.